### DISCUSSION TITLE
Handling invalid documents

### DISCUSSION THREAD

**Discussion by nkoudelia** [phax/phase4#282] [Q&A] [answered]
I'm getting lots of errors like this:

```text
[error] [invalid-standard] in [SBDH/DocumentIdentification/Standard] The 'DocumentIdentification/Standard' element has the illegal value 'urn:oasis:names:specification:ubl:schema:xsd:Invoice-2'. It must equal the namespace URI of the business message ('urn:oasis:names:specification:ubl:schema:xsd:CreditNote-2'). It must also be the beginning of the document type identifier value ('urn:oasis:names:specification:ubl:schema:xsd:Invoice-2::Invoice##urn:cen.eu:en16931:2017#compliant#urn:fdc:peppol.eu:2017:poacc:billing:3.0#conformant#urn:fdc:anskaffelser.no:2019:ehf:forward-billing:3.0::2.2').
[error] [invalid-type] in [SBDH/DocumentIdentification/Type] The 'DocumentIdentification/Type' element has the invalid value 'Invoice'. It must match the local name of the root element of the business message ('CreditNote').
```
The errors are self-explanatory and incoming documents are indeed invalid: the SBDH DocumentIdentification states it's an Invoice but the nested XML document is actually a CreditNote.

Oxalis doesn't stop such documents and hence ~~many~~ some participants still have those. Is it possible to pass those with a warning?

**Comment by phax** [comment, marked as answer]
In this case, the error must be fixed in Oxalis.
Changing errors to warnings is bad style.
Best practise may be, that you create an respective issue at the correct Oxalis project (evtl. https://github.com/OxalisCommunity/oxalis), if it bugs you.
